shared/config/ports.js

```diff
@@ -2,5 +2,5 @@
  * Review application ports
  */
 module.exports = {
-  app: process.env.PORT || 3000
+  app: process.env.PORT || 8080
```

Comment:
We've moved the Review app to port `:8080` but it could still conflict here

Open to other suggestions